app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -350,6 +354,7 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
                 }
                 // Make sure out sync engine checkboxes are up-to-date.
                 updateSyncEngineStates()
+                setEnginesDisabledWhileSyncing(false)
```

Comment:
It looks like we're intentionally leaving the keys disabled in `onError` (by not calling `setEnginesDisabledWhileSyncing(false)`).  Is that correct?  That makes sense to me (we can't do anything with their clicks until they reconnect anyway), but would you mind adding a comment in `onError` that states it explicitly?